code/game/machinery/doors/brigdoors.dm

```diff
@@ -99,7 +97,7 @@
 			rank = I.assignment
```

Comment:
At 92: the R.fields ["criminal"] = SEC_RECORD_STATUS_INCARCERATED, pro "Incarcerated" - apologies, this is my bad since I missed to change these, while making the statuses as defines.

Reply:
Fixed